src/status_im/init/core.cljs

```diff
@@ -78,13 +85,14 @@
 (fx/defn initialize-app-db
   "Initialize db to initial state"
   [{{:keys [status-module-initialized? view-id hardwallet
-            initial-props
+            initial-props desktop/desktop
             network-status network peers-count peers-summary device-UUID]
      :node/keys [status]
      :or   {network (get app-db :network)}} :db}]
   {:db (assoc app-db
               :contacts/contacts {}
               :initial-props initial-props
+              :desktop/desktop (merge (:desktop/desktop app-db) desktop)
```

Comment:
I'm probably missing something, where does `desktop` come from?

Reply:
It comes from `desktop/desktop` above in `:keys` section of fn's arg list (https://clojure.org/guides/destructuring#_namespaced_keywords)

Reply:
Gotcha, I thought you needed to reference it as `desktop/desktop`. Thanks for the link!